ecs-cli/modules/cli/compose/project/project_parseV3.go

```diff
@@ -11,20 +17,93 @@ import (
 func (p *ecsProject) parseV3() (*[]containerconfig.ContainerConfig, error) {
 	logrus.Debug("Parsing v3 project...")
 
-	// TODO: parse v3, convert ServiceConfigs to ContainerConfigs
-	configDetails := types.ConfigDetails{}
+	// load and parse each compose file into a configFile
+	configFiles := []types.ConfigFile{}
+	for _, file := range p.ecsContext.ComposeFiles {
+
+		loadedFile, err := ioutil.ReadFile(file)
+		if err != nil {
+			return nil, err
+		}
+		parsedFile, err := loader.ParseYAML(loadedFile)
+		if err != nil {
+			return nil, err
+		}
+		configFile := types.ConfigFile{
+			Filename: file,
+			Config:   parsedFile,
+		}
+		configFiles = append(configFiles, configFile)
+	}
+
+	// get build environment
+	env, err := getLocalEnv()
+	if err != nil {
+		return nil, err
+	}
+
+	// get compose working dir
+	wrkDir, err := getWorkingDir(p.ecsContext.ComposeFiles[0])
+	if err != nil {
+		return nil, err
+	}
+
+	// build config details
+	configDetails := types.ConfigDetails{
+		WorkingDir:  wrkDir,
+		ConfigFiles: configFiles,
+		Environment: env,
+	}
+
+	// load config from config details
 	config, err := loader.Load(configDetails)
 	if err != nil {
 		return nil, err
 	}
 
+	// convert ServiceConfigs to ContainerConfigs
+	conConfigs := []containerconfig.ContainerConfig{}
 	for _, service := range config.Services {
-		convertDockerToContainerConfig(service)
+		cCon, err := convertDockerToContainerConfig(service)
+		if err != nil {
+			return nil, err
+		}
+		conConfigs = append(conConfigs, *cCon)
 	}
 
-	return &[]containerconfig.ContainerConfig{}, nil
+	// return container configs
+	return &conConfigs, nil
+}
+
+func convertDockerToContainerConfig(serviceConfig types.ServiceConfig) (*containerconfig.ContainerConfig, error) {
+	//TODO: fully convert ServiceConfig to ContainerConfig
+	c := &containerconfig.ContainerConfig{
+		Name: serviceConfig.Name,
+	}
+	return c, nil
 }
 
-func convertDockerToContainerConfig(serviceConfig types.ServiceConfig) {
-	return
+func getWorkingDir(fileName string) (string, error) {
+	if strings.Index(fileName, "/") != 0 {
+		workDir, err := os.Getwd()
+		if err != nil {
+			return "", errors.Wrap(err, "Unable to retrieve compose file directory")
+		}
+		fileName = filepath.Join(workDir, fileName)
+	}
+	return filepath.Dir(fileName), nil
+}
+
+func getLocalEnv() (map[string]string, error) {
+	osEnv := os.Environ()
+	envMap := make(map[string]string)
+
+	for _, envString := range osEnv {
+		if !strings.Contains(envString, "=") {
```

Comment:
Hm, would this realistically ever happen? From what I can tell, `os.Environ()` makes a system call to get the env variables on a given system. 

More to the point though, do we actually only care about a particular set of environment variables? I assumed that maybe was the case on reading the error message, which is why validating only on having the string contain an `=` was a bit confusing to me.